test/dispatchers/dispatcherTests.ts

```diff
@@ -115,11 +115,11 @@ describe("Dispatchers", () => {
       var md = Plottable.Dispatcher.Mouse.getDispatcher(<SVGElement> target.node());
 
       var cb1Called = false;
-      var cb1 = function(p: Plottable.Point) {
+      var cb1 = function(p: Plottable.Point, e: MouseEvent) {
         cb1Called = true;
       };
       var cb2Called = false;
-      var cb2 = function(p: Plottable.Point) {
```

Comment:
You can make the callbacks
var cb1 = () => cb1Called = true;
and the same for cb2